packages/lime-proto-olsr/src/olsr.lua

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/lua
+
+local network = require("lime.network")
+local config = require("lime.config")
+local fs = require("nixio.fs")
+local libuci = require("uci")
+local wireless = require("lime.wireless")
+local utils = require("lime.utils")
+
+olsr = {}
+
+olsr.configured = false
+
+function olsr.configure(args)
+	if olsr.configured then return end
+	olsr.configured = true
+
+	local uci = libuci:cursor()
+	local ipv4 = network.primary_address()
+
+	fs.writefile("/etc/config/olsrd", "")
+
+	uci:set("olsrd", "lime", "olsrd")
+	uci:set("olsrd", "lime", "LinkQualityAlgorithm", "etx_ff")
+	uci:set("olsrd", "lime", "IpVersion", "4")
+
+    uci:set("olsrd", "limejson", "LoadPlugin")
```

Comment:
Seems the indentation is not constant, please use tabulator [printf("\t")] as indentation character everywhere